api/agent/uniter/unit_test.go

```diff
@@ -909,6 +898,7 @@ func (s *unitSuite) TestWatchAddressesHashNotImplemented(c *gc.C) {
 }
 
 func (s *unitSuite) TestRelationStatus(c *gc.C) {
+	c.Skip("rewrite test with Relations epic")
```

Comment:
Just delete the test and add in plain words what the test was doing so we can add it back in the future.

Reply:
This is the wrong comment. The test will not be rewritten as this is api not apiserver code. It's commented out while for the bit of time where RelationsStatus and WatchUnits is disabled.